pkg/config/template.go

```diff
@@ -40,24 +34,16 @@ const (
 )
 
 func getTemplates(templatePath string) (CiTemplates, error) {
-	templates := make(map[string]*templateapi.Template)
+	templates := make(CiTemplates)
 	err := filepath.Walk(templatePath, func(path string, info os.FileInfo, err error) error {
-		if err != nil {
-			return fmt.Errorf("prevent panic by handling failure accessing a path %q: %v", path, err)
+		if err != nil || info.IsDir() || !(strings.HasSuffix(path, ".yaml") || strings.HasSuffix(path, ".yml")) {
```

Comment:
this check can be done with `isYAML()` as well.
```go
if err != nil || isYAML(path, info) {
```

Reply:
I inlined that function because it is only ever used here, but that got lost when I moved commits around. Updated.

Reply:
why we want `(strings.HasSuffix(path, ".yaml")` to be false?